pkg/apis/apps/v1beta1/register.go

```diff
@@ -57,6 +57,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&Scale{},
 		&StatefulSet{},
 		&StatefulSetList{},
+		&ControllerRevision{},
```

Comment:
These APIs will technically be alpha, we should probably put them in an alpha group (@lavalamp)